R/PlainTextParam.R

```diff
@@ -0,0 +1,215 @@
+#' @title Store `MsExperiment` and `XcmsExperiment` object as a folder of plain
+#' text files
+#'
+#' @name PlainTextParam
+#'
+#' @export
+#'
+#' @family xcms result export formats.
+#'
+#' @description
+#' The `PlainTextParam` class and method enable users to save `MsExperiment` or
+#' `XcmsExperiment` objects as collections of plain text files in a specified
+#' folder.
```

Comment:
Maybe also add here a note that exporting `MsExperiment` currently only works with `MsBackendMzR`? Something like:

```
... folder. Note that, while for all xcms results within the `XcmsExperiment` can and will be exported, 
the full raw MS data (of the object's `Spectra` object) will currently not be exported in plain text format. 
For `Spectra` using the [MsBackendMzR()] backend, the names of the raw data files will however be 
exported (which enables to *restore* the full `Spectra` respectively `MsExperiment` objects).
```